src/components/List/AddCard/AddCardForm/AddCardForm.js

```diff
@@ -53,11 +53,11 @@ const AddCardForm = ({ setOpen, open, listId }) => {
         value={text}
         placeholder="Type something..."
         autoFocus
-        className="w-full box-border outline-none border-2 border-trello-blue-100 shadow bg-white hover:bg-trello-gray-400 rounded-md p-2 my-1.5"
```

Comment:
You've gotten rid of the border here. Any reason why?

Reply:
Yes, I am referencing Trello's design, when adding a new card there is no border around the add card input.

Reply:
Good stuff, in that case, could you also remove `hover:bg-trello-gray-400` from that as well so you dont get a weird hover effect